### PR TITLE
feat(TMRX-1312): Add tracking to Slice Header

### PR DESCRIPTION
**Pull request by bibicollins1**
### Description

Add tracking to Slice Header component.

[TMRX-1312](https://nidigitalsolutions.jira.com/browse/TMRX-1312)

### Checklist

- [x] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

[TMRX-1312]: https://nidigitalsolutions.jira.com/browse/TMRX-1312?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ts-newskit/src/slices/fixtures/analytics-actions.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { action } from '@storybook/addon-actions';
+
+export default (event: any | any[]) => {
```
Comment (yukiinasawa) [inline packages/ts-newskit/src/slices/fixtures/analytics-actions.ts:3]: Just for own understanding - if event could be an array then we need to map to call each action?

Comment (bibicollins1) [reply packages/ts-newskit/src/slices/fixtures/analytics-actions.ts:3]: Oh I think that's from the original component which also handled scroll interactions 
Can remove that type from this

Comment (bibicollins1) [reply packages/ts-newskit/src/slices/fixtures/analytics-actions.ts:3]: Do have a point though, in this case would just be passing the single event so don't need the array type

Comment (yukiinasawa) [reply packages/ts-newskit/src/slices/fixtures/analytics-actions.ts:3]: Ah okay - so `ts-components` is handling both cases. - thanks 👍

File: packages/ts-newskit/package.json [hunk 2]
```diff
@@ -54,11 +54,13 @@
     "@emotion/styled": "11.11.0",
     "@newskit-themes/the-times": "0.1.1",
     "isomorphic-unfetch": "3.1.0",
+    "lodash.merge": "4.6.2",
```
Comment (yukiinasawa) [inline packages/ts-newskit/package.json:57]: Are we using this somewhere in this PR?

**Review by yukiinasawa — APPROVED** [review]

**Review by ibrahim-kurhan — APPROVED** [review]

**Pull request merged**